tests/runtime/json-output

```diff
@@ -72,102 +72,102 @@ TIMEOUT 5
 
 NAME printf
 RUN {{BPFTRACE}} -q -f json -e 'BEGIN { printf("test %d", 5); exit(); }'
-EXPECT ^{"type": "printf", "data": "test 5"}$
+EXPECT {"type": "printf", "data": "test 5"}
```

Comment:
These should all be EXPECT_JSON (as that tolerates different json field ordering) and we should either update EXPECT_JSON to support raw strings instead of file names or just maybe add EXPECT_JSON_FILE - I can do that in a follow-up PR if you don't want to do it here.

Reply:
Good idea, but I'm afraid the EXPECT fatigue has got me too so I'll leave it out of this PR 😅